### DISCUSSION TITLE
Error in Geany try to execute

### DISCUSSION THREAD

**Discussion by sergio-rms** [geany/geany#3181] [General]
After a windows atualization , my Geanie doesn´t work anymore... I receive this message : 12:03:41: Esse é o Geany 1.38.

12:04:56: Impossível executar o comando de compilação "cmd.exe /Q /C ^"C:\Program^ Files\Geany\libexec\geany\geany-run-helper^"^ ^"C:\python_works^"^ 0^ py^ ^"w.py^"": Não foi possível concluir a operação com êxito porque o arquivo contém um vírus ou software possivelmente indesejado. Verifique a definição do terminal nas Preferências

**Comment by elextr** [comment]
Your updated Windows virus checker is most likely providing a false positive on `geany-run-helper.bat` or Python.  The batch file is text so you can check its not a virus, but I don't know how to tell Windows/Virus checker that its ok.  Python is not part of Geany and needs to be addressed by that project.

**Comment by sergio-rms** [comment]
Thank you very much for the insight ! Really, some software installed without permission an antivirus called RAV , and after suffering a little to remove it , Geany returned to work very well ! God bless you!


Sérgio Morato
Gerente Serviços
[Semapi Manutenção e Suprimentos]<https://www.semapi.com.br/>
Fone: 11-2144-4300
***@***.******@***.***>
www.semapi.com.br <https://www.semapi.com.br>
[Facebook icon]<https://www.facebook.com/semapisolucoes/>   [LinkedIn icon] <https://www.linkedin.com/company/semapisolucoes/>    [Instagram icon] <https://www.instagram.com/semapisolucoes/>


De: ***@***.***>
Enviado:segunda-feira, 25 de abril de 2022 20:04
Para: ***@***.***>
Cc:Sérgio R.M. ***@***.***>; ***@***.***>
Assunto: Re: [geany/geany] Error in Geany try to execute (Discussion #3181)


Your updated Windows virus checker is most likely providing a false positive on geany-run-helper.bat or Python. The batch file is text so you can check its not a virus, but I don't know how to tell Windows/Virus checker that its ok. Python is not part of Geany and needs to be addressed by that project.

—
Reply to this email directly, view it on GitHub<https://github.com/geany/geany/discussions/3181#discussioncomment-2634526>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AY4KIZOZMIP5YHWMCWGDKW3VG4QGNANCNFSM5UI4Y43A>.
You are receiving this because you authored the thread.Message ID: ***@***.***>


Esta mensagem, bem como seus anexos, é destinada exclusivamente ao seu destinatário e pode conter informações confidenciais, protegidas por sigilo profissional ou cuja divulgação seja proibida por lei. Informações contidas nesta mensagem, não expressam obrigatoriamente a posição oficial da Semapi. O uso não autorizado de tais informações é proibido e está sujeito às penalidades cabíveis.

This message and its attachments is intended exclusively for its addressee and may contain information that is confidential and protected by a professional privilege or whose disclosure is prohibited by law. Information contained in this message and do not necessarily express the official position of Semapi. Unauthorized use of such information is prohibited and subject to applicable penalties.



________________________________
Esta mensagem, bem como seus anexos, é destinada exclusivamente ao seu destinatário e pode conter informações confidenciais, protegidas por sigilo profissional ou cuja divulgação seja proibida por lei. Informações contidas nesta mensagem, não expressam obrigatoriamente a posição oficial da Semapi. O uso não autorizado de tais informações é proibido e está sujeito às penalidades cabíveis.

This message and its attachments is intended exclusively for its addressee and may contain information that is confidential and protected by a professional privilege or whose disclosure is prohibited by law. Information contained in this message and do not necessarily express the official position of Semapi. Unauthorized use of such information is prohibited and subject to applicable penalties.
________________________________
